docs/tutorials/lets_write_a_test.md

```diff
@@ -1,74 +1,81 @@
 ## The Task
 
-Consider if mcollectived incorrectly spawned a new process with every puppet agent run on Ubuntu 10.04.  We need an acceptance test to check that a new process is not spawned and to ensure that this issue does not regress in new builds.
+We will write an acceptance test to check if a package (specifically HTTPD) is installed and running
 
 ## Figure Out Test Steps
 
 What needs to happen in this test:
 
-* Install PE
-* Restart mcollective twice
-* Check to see if more than one mcollective process is running
+* Install and run httpd
+  * Test that HTTPD is not installed
+  * Install HTTPD
+  * Start HTTPD service
+* Testing
+  * Test HTTPD is installed
+  * Test HTTPD is running
 
 ## Create a host configuration file
     $ beaker-hostgenerator redhat7-64ma > redhat7-64ma.yaml
 
-## Install PE
+## Install and run HTTPD
 
-We prefer to install PE once and then run a set of tests, so PE installation should not be part of the actual acceptance test.
+Make a file named `install.rb` and put the following code into it:
 
-    $ mkdir setup
-    $ cd setup
-    $ cat > install.rb << RUBY
-    test_name "Installing Puppet Enterprise" do
-    install_pe
-    RUBY
-    $ cd ..
-
-This places our install steps in a ruby script (install.rb) which will run on localhost, but the #install_pe method knows where to install puppet enterprise based on the host configuration in use.
-The install.rb script is used in our commandline to beaker, below.
-
-## Create a test file
-
-We need to create a test file to run.
-
-### Define some test commands to run
-#### Restart mcollective twice
-
-Here's our magic command that restarts mcollective:
-
-    restart_command = "bash -c '[[ -x /etc/init.d/pe-mcollective ]] && /etc/init.d/pe-mcollective restart'"
-
-#### Check to see if more than one mcollective process is running
+```ruby
+test_name "Installing and runnning HTTPD" do
+
+	step "Make sure HTTPD is not installed" do
+	  hosts.each do |host|
+		# Check if HTTPD is not installed
+		assert_equal(false, check_for_package(host, 'httpd'))
```

Comment:
It doesn't seem correct that this will fail if `httpd` is already installed; I'd rather see that this test just ensure that httpd is installed, not fail if `httpd` is already available on an SUT.